db/migrate/20210908220918_create_status_edits.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStatusEdits < ActiveRecord::Migration[6.1]
+  def change
+    create_table :status_edits do |t|
+      t.belongs_to :status, null: false, foreign_key: { on_delete: :cascade }
+      t.belongs_to :account, null: false, foreign_key: { on_delete: :nullify }
```

Comment:
`on_delete: nullify` and `null: false`, this seems like an error

Reply:
Good catch...